crates/claims/crates/status/src/impl/token_status_list/mod.rs

```diff
@@ -437,3 +564,150 @@ impl<'a> StatusMapEntry for AnyStatusListReference<'a> {
         }
     }
 }
+
+#[cfg(test)]
```

Comment:
same here -- can you add test cases for out-of-bounds access & value-setting attempts?